consensus/config/src/parameters.rs

```diff
@@ -14,14 +14,25 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct Parameters {
     /// Time to wait for parent round leader before sealing a block.
+    /// Default: 250ms
```

Comment:
Although those are useful when reading I believe they can easily go obsolete if someone changes the default values and don't come back to fix it. Not a big issue, but we might want to avoid having those comments and rely entirely in code as source of truth

Reply:
Makes sense. Removed.